### ISSUE TITLE
[Question]: 使用DEV分支finetune.py进行多卡微调时报错

### ISSUE THREAD

**Issue by LANSEYOUYUMOWANG** [PaddlePaddle/PaddleNLP#4168]
### 请提出你的问题

代码为dev分支的最新代码
paddlenlp 的引入的版本为：2.4.5
![8233ff2240038178ac7dddb77c1417e](https://user-images.githubusercontent.com/36221128/208602906-6f096ce4-09f0-4552-8b70-70c4f5257989.png)
使用启动多卡微调的命令如下：
python -u -m paddle.distributed.launch --gpus "0,1" finetune.py --device gpu --logging_steps 10 --save_steps 100 --eval_steps 100 --seed 42 --model_name_or_path uie-base --output_dir ./data//checkpoint/model_best --train_path data/train.txt --dev_path data/dev.txt --max_seq_length 512 --per_device_eval_batch_size 16 --per_device_train_batch_size  16 --num_train_epochs 100  --learning_rate 1e-5 --do_train --do_eval --do_export  --export_model_dir ./checkpoint/model_best --label_names 'start_positions' 'end_positions' --overwrite_output_dir --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end  True --save_total_limit 1 
执行后报如下错误：
![3c99a102d011ed2f24f1b8bbfc36731](https://user-images.githubusercontent.com/36221128/208603421-9cdfd5c6-bb2a-4404-a470-5fecdfdf6041.png)

**Comment by linjieccc** [comment]
@LANSEYOUYUMOWANG Hi，

需要安装一下GPU版本的paddlepaddle：https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/linux-pip.html

**Comment by LANSEYOUYUMOWANG** [comment]
按照回复安装GPU版本的paddlepaddle：https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/linux-pip.html
![image](https://user-images.githubusercontent.com/36221128/209045016-b0520e19-677e-4388-ba88-09d11a4fd124.png)
报如下错误：
![bc920725ceda45569cd5f8d4eff19ca](https://user-images.githubusercontent.com/36221128/209045063-9574c757-bd90-400a-be3d-b311792a6b7d.png)

**Comment by wawltor** [comment]
> 按照回复安装GPU版本的paddlepaddle：https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/linux-pip.html ![image](https://user-images.githubusercontent.com/36221128/209045016-b0520e19-677e-4388-ba88-09d11a4fd124.png) 报如下错误： ![bc920725ceda45569cd5f8d4eff19ca](https://user-images.githubusercontent.com/36221128/209045063-9574c757-bd90-400a-be3d-b311792a6b7d.png)

安装有问题，建议conda安装，如果conda安装不行
1）回退到2.4.0rc版本
2）使用docker容器

**Comment by LANSEYOUYUMOWANG** [comment]
按照回复内容，使用docker容器，paddlepaddle-gpu 版本回退到2.4.0rc0
![image](https://user-images.githubusercontent.com/36221128/210320559-cc8bb5a8-8368-4bdb-886d-07d3a2f8a4bf.png)
使用如下命令进行多卡微调：
python -u -m paddle.distributed.launch --gpus "0,1" finetune.py   --device gpu  --logging_steps 10 --save_steps 100 --eval_steps 100  --seed 42  --model_name_or_path uie-base --output_dir ./checkpoint/model_best --train_path data/train.txt --dev_path data/dev.txt --max_seq_length 512  
会导致receive shutdown event and so quit from MasterDaemon run loop直接结束微调

![image](https://user-images.githubusercontent.com/36221128/210320910-565fb11a-6024-412c-9fe7-7450f526b7bd.png)

**Comment by LANSEYOUYUMOWANG** [comment]
换了高算力的显卡，该问题解决
